referencia/webpayrest/README.md

```diff
@@ -311,55 +282,37 @@ token es caducado y no podrá ser utilizado en un pago.
 </aside>
 
 ```java
-import cl.transbank.webpay.exception.CreateTransactionException;
-import cl.transbank.webpay.webpayplus.WebpayPlus;
-import cl.transbank.webpay.webpayplus.model.CreateWebpayPlusTransactionResponse;
-import java.util.Random;
-public class IntegrationExample {
-    public static void main(String[] args) {
-        String buyOrder = String.valueOf(new Random().nextInt(Integer.MAX_VALUE));
-        String sessionId = String.valueOf(new Random().nextInt(Integer.MAX_VALUE));
-        double amount = 1000;
-        String returnUrl = "https://domai.cl/webpay-return";
-        try {
-            final WebpayPlusTransactionCreateResponse response = WebpayPlus.Transaction.create(buyOrder, sessionId, amount, returnUrl);
-            final String token = response.getToken();
-            final String url = response.getUrl();
-        } catch (TransactionCreateException | IOException e) {
-            e.printStackTrace();
-        }
-    }
-}
+final WebpayPlusTransactionCreateResponse response = WebpayPlus.Transaction.create(
+  buyOrder, sessionId, amount, returnUrl
+);
 ```
 
 ```php
-use Transbank\Webpay\WebpayPlus\Transaction;
```

Comment:
I think there are more than one Transaction Object on the SDK, so we should not delete de entire namespace to be specific

Reply:
Sounds good